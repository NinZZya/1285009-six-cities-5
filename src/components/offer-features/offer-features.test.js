import React from 'react';
import renderer from 'react-test-renderer';
import OfferFeatures from './offer-features';
import OfferFeature from './components/offer-feature/offer-feature';

const Feature = {
  ENTIRE: {
    type: `entire`,
    value: `Private room`,
  },
  BEDROOMS: {
    type: `bedrooms`,
    value: `2 Bedrooms`,
  },
  ADULTS: {
    type: `adults`,
    value: `Max 3 adults`,
  },
};


it(`Should OfferFeatures render correctly`, () => {
  const tree = renderer
    .create(
        <OfferFeatures>
          <OfferFeature type={Feature.ENTIRE.type} value={Feature.ENTIRE.value} />
          <OfferFeature type={Feature.BEDROOMS.type} value={Feature.BEDROOMS.value} />
          <OfferFeature type={Feature.ADULTS.type} value={Feature.ADULTS.value} />
        </OfferFeatures>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
