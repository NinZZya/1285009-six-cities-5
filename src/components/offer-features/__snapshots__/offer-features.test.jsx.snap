// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should OfferFeatures render correctly 1`] = `
<ul
  className="property__features"
>
  <li
    className="property__feature property__feature--entire"
  >
    Private room
  </li>
  <li
    className="property__feature property__feature--bedrooms"
  >
    2 Bedrooms
  </li>
  <li
    className="property__feature property__feature--adults"
  >
    Max 3 adults
  </li>
</ul>
`;
